assets/static/js/index.js

```diff
@@ -0,0 +1,22 @@
+const urlParams = new URLSearchParams(window.location.search);
+
+document.addEventListener("DOMContentLoaded", function () {
+  // leverage the URL parameter provided from the request and set it to the target origin
+  const encodedUriComponent = urlParams.get("origin");
+  if (!encodedUriComponent) {
+    alert("An origin URL parameter must be provided.")
```

Comment:
We should add a todo as a P3-4 to make this better for the UX, alerts arent the friendliest, but certainly shouldnt stop initial implementation.

Reply:
ack ill add it to the design doc, as well as some of the other non blocking requests from this PR i.e. css linter in pkg, browser support matrix,